### DISCUSSION TITLE
Save GP Radio and Live Timing positions in layout

### DISCUSSION THREAD

**Discussion by zflash** [f1multiviewer/issue-tracker#5] [Feature requests] [answered]
The save layout option is great, but it doesn't add GPRadio in it when saving a layout. Could be handy to save it as well in the layout option so that it automatically opens when loading a layout.

**Comment by f1multiviewer** [comment]
This is likely a duplicate of https://github.com/f1multiviewer/issue-tracker/discussions/7 (at least partially)

**Reply by f1multiviewer** [reply]
Or the other way around :')

**Reply by kidforpm** [reply]
Close one, and combine the votes. I think this is a popular one.

**Comment by ExtremTechniker** [comment]
Is this still on the ToDo list?

**Comment by f1multiviewer** [comment, marked as answer]
This is now done (GP radio excluded as it's potentially going to be deprecated soon, and not really part of F1)
